numpy/lib/tests/test_packbits.py

```diff
@@ -17,6 +17,22 @@ def test_packbits():
     assert_raises(TypeError, np.packbits, np.array(a, dtype=float))
 
 
+def test_packbits_empty():
+    for dtype in [np.bool, np.uint8, np.int]:
```

Comment:
Also not a good idea to shadow the numpy `dtype` variable. Use something like `dt` instead.